doc/editor-integration.md

```diff
@@ -228,8 +228,8 @@ and CLJC (`.cljc`)<sup>1</sup>.
 4. Scope: `Current file`
 5. In the Program field, type `clj-kondo`
 6. In the Arguments field, type `--lint $FilePath$`<br>
-You may use a custom config E.g `--lint $FilePath$ --config "{:lint-as {manifold.deferred/let-flow clojure.core/let}}"`
-7. In the Working directory field, type `$FileDir$`
+You may use a custom config E.g `--lint $FilePath$ --config "{:lint-as {manifold.deferred/let-flow clojure.core/let}}"` or `--lint $FilePath$ --config your-custom-path/clj-kondo.conf.js"`
+7. In the Working directory field, type `$ProjectFileDir$`
```

Comment:
In case of a mono-repo what is the `$ProjectFileDir` in Cursive? Using `FileDir` clj-kondo will see configs at a higher level than the root directory of the mono-repo.

Reply:
Hard to say about monorepo, I don't have any examples. If I understand your doubts correctly each project inside monorepo could have its own clj-kondo config and IntelliJ IDEA default macro `$FileDir$` will find the closest one and you would like to keep this behavior.
If it is the main concerns we can just leave an example with a custom file config and remove step 7 as it is the default IntelliJ IDEA behavior
If this PR has no sense at all I'm fine to close it as well

Reply:
In the only monorepo project I work on it points to the root of the monorepo. I have a deps.edn there and have deps.edns in child folders (subprojects).

Reply:
Then it is better to leave it as is.